src/plugins/google/ssh-key.ts

```diff
@@ -0,0 +1,75 @@
+/** Copyright © 2024-present P0 Security
+
+This file is part of @p0security/cli
+
+@p0security/cli is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, version 3 of the License.
+
+@p0security/cli is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with @p0security/cli. If not, see <https://www.gnu.org/licenses/>.
+**/
+import { asyncSpawn } from "../../common/subprocess";
+import { print2 } from "../../drivers/stdio";
+import { ImportSshPublicKeyResponse } from "./types";
+
+/**
+ * Adds an ssh public key to the user object's sshPublicKeys array in Google Workspace.
+ * GCP OS Login uses these public keys to authenticate the user.
+ * Importing the same public key multiple times is idempotent.
+ *
+ * The user account and the access token is retrieved from the gcloud CLI.
+ *
+ * Returns the posix account to use for SSH access.
+ *
+ * See https://cloud.google.com/compute/docs/oslogin/rest/v1/users/importSshPublicKey
+ */
+export const importSshKey = async (
+  publicKey: string,
+  options?: { debug?: boolean }
```

Comment:
What if you just made debug required? `options : {debug: boolean} = {debug: false}`.

Then you could pass the options into async spawn directly: `await asyncSpawn(options, "gcloud", [...]`